particlespy/seg_ui.py

```diff
@@ -381,6 +380,7 @@ def getim(self,im_hs):
         im = im-np.min(im)
         image = np.uint8(255*im/np.max(im))
         self.image = image
+        print(image.shape)
```

Comment:
Remove?

Reply:
the new commit should resolve both of these issues, I've double checked manual segmentation and it works fine too